concordium-std/src/impls.rs

```diff
@@ -521,6 +526,18 @@ pub fn put_in_memory(input: &[u8]) -> *mut u8 {
     ptr
 }
 
+/// Wrapper for HasActions::Send, which automatically serializes the parameter.
+#[inline(always)]
```

Comment:
I'm not sure that the inline here is a good idea.

We'd need to do some measurements to see whether it makes sense.